src/robot/running/testlibraries.py

```diff
@@ -91,13 +91,17 @@ def doc(self):
         return self._doc
 
     @property
-    def listener(self):
+    def listeners(self):
         if self.has_listener:
-            return self._get_listener(self.get_instance())
+            return self._get_listeners(self.get_instance())
         return None
 
-    def _get_listener(self, inst):
-        return getattr(inst, 'ROBOT_LIBRARY_LISTENER', None)
+    def _get_listeners(self, inst):
+        try:
+            listeners = getattr(inst, 'ROBOT_LIBRARY_LISTENER')
+            return listeners if isinstance(listeners, list) else [listeners]
+        except AttributeError:
```

Comment:
I always find it a bit risky to catch `AttributeError` when there's more than one line of code. It's so easy to trigger this exception by just having a typo or changing APIs elsewhere so that an attribute is removed. That's not likely here, but I'd still prefer having `try/except AttributeError` just around `getattr`. It might be even better to use `hasattr`, though:

```python
if not hasattr(inst, 'ROBOT_LIBRARY_LISTENER'):
    return None
listeners = inst.ROBOT_LIBRARY_LISTENER
# ...
```